### ISSUE TITLE
Dev server does not serve paths with extension

### ISSUE THREAD

**Issue by kirill-konshin** [insin/nwb#304]
This issue is a: Bug report

- [x] Bug report
- [ ] Feature request
- [ ] Question / support request
- [ ] Other

Request to path like `/xxx.html` or `/xxx/yyy.html` results in `Cannot GET /xxx.html` error. If `.html` will be removed everything works as expected.

Could be related to https://github.com/webpack/webpack-dev-server/issues/332.

**Comment by kirill-konshin** [comment]
I tried to do the following, but it did not help:

```js
module.exports = function() {
    return {
        webpack: {
            extra: {
                devServer: {
                    historyApiFallback: {
                        disableDotRule: true
                    }
                }
            }
        }
    };
};
```

I also tried `rewrites` from the above-mentioned link instead of `disableDotRule`, seems that `extra.devServer` is ignored...

**Comment by insin** [comment]
You couldn't configure this before because nwb was running its own dev server, but as of 21f57ba it's now using webpack-dev-server, which can be configured with top-level `devServer` config:

```js
module.exports = {
  devServer: {
    historyApiFallback: {
      disableDotRule: true
    }
  }
}
```

This will be fixed when `next` is merged back into `master` for release.

**Comment by insin** [comment]
This should be fixed in v0.17
